### DISCUSSION TITLE
Undo Pages unpublish

### DISCUSSION THREAD

**Discussion by joshuaneman** [community/community#28190] [Other Feature Feedback, Questions, & Ideas] [answered]
Is it possible to re-publish a Pages site? I followed the instructions [here to unpublish](https://docs.github.com/en/pages/getting-started-with-github-pages/unpublishing-a-github-pages-site), but there's no documentation on undoing that or republishing. 

![image](https://user-images.githubusercontent.com/39058400/182628287-5e3fb7bd-5398-4538-bdd8-1388cbe186f9.png)

**Comment by devtayyab** [comment]
https://docs.github.com/en/enterprise-server@3.4/pages/getting-started-with-github-pages/unpublishing-a-github-pages-site

you can check it

**Comment by joshuaneman** [comment]
@devtayyab Thanks for the link. I didn't see on that page how to re-able pages after unpublishing, though.

**Comment by jfullstackdev** [comment]
check the settings again, the same thing you did when you enabled it for the first time, I guess

**Comment by gergp62** [comment]
I have the same problem too, I can´t find on docs how to re publish the site again after some changes...maybe after some time the option save appears again...

**Reply by gergp62** [reply]
I fix it...try to select branch none, save then enter in the repository- setting-pages and select branch master, save and that´s it, the link of the live site appears again....try it!

**Reply by VATSAL77726** [reply]
Thank You.

**Reply by Pawaomoro** [reply]
thanks mahn it worked

**Comment by joshuaneman** [comment, marked as answer]
I reached out to GitHub support and got the following guidance: 

>  You can re-publish your GitHub Pages website by navigating to the Actions tab of your repository, click on the Pages build and deployment workflow, then click the Re-run all jobs button located in the upper right corner of the page. After the workflow run has completed, your site will be published.

**Reply by DonyMaMUST** [reply]
but I did not find Re-run all jobs button located in the upper right corner of the page, what should I do?555555

**Reply by MArrasmith** [reply]
you have to select the actual work flow hyperlink of the action, and then on the following page it will populate with the re-run all jobs button in the top right corner

**Reply by Kessi-ux** [reply]
Please can you explain with pictures

**Reply by x3lna** [reply]
thank you so much this helped me a lot

**Reply by Mi-Da-Da** [reply]
Thank you so much for providing this advice!I'm so worried that it may can't be republish again.Your advice helped me a lot.

**Reply by Mi-Da-Da** [reply]
> but I did not find Re-run all jobs button located in the upper right corner of the page, what should I do?555555但是我没有找到位于页面右上角的 Re-run all jobs 按钮，我该怎么办 555555

I just clicked on the Pages build and deployment workflow and the site was published again.

**Comment by ligang19999** [comment]
you can go to settings → General → Danger Zone, change your repository visibility to private, and change to public again, then you go to settings → pages, and you can deploy your page again, and your site will turn up again

**Reply by Noxisrambler30** [reply]
Thank you soo much

**Reply by Akshat002r** [reply]
Thankyou So much for the Help.

**Reply by Opakeeperz** [reply]
thank you i forgot to add a <a></a> element to my home button so THANKS
btw the website is [this markdown link](opakeeperz.github.io/lamasum/)

**Reply by parmarkamalesh** [reply]
Thanks, It worked.

**Reply by piskwire** [reply]
Thanks

**Reply by pansarasara** [reply]
thank you. this works for me. but I also have to change the default file name static..yml. if not it would show message "file with same name alredy exist"
